### PR TITLE
Change over to required `gql` tag

### PR DESCRIPTION
**Pull request by stubailo**
This will make it required to wrap all queries with:

```js
gql`...`
```

See docs PR for details: https://github.com/apollostack/docs/pull/63

TODO:

- [x] Test that appropriate warnings are printed
- [x] Update change log
- [x] Export `gql` from a subdirectory
- [x] Use `npm link` to test from an app
- [x] Set up PR to docs

### CODE REVIEW

**Comment by jbaxleyiii** [comment]
> Use npm link to test from an app

I haven't tested that yet and it *probably* won't work? Still need to figure this one out

**Comment by stubailo** [comment]
It will work from this branch though since it's before your other changes! Definitely have used that before.

**Comment by jbaxleyiii** [comment]
Oh cool! If that is the case it should keep working. The deploy branch doesn't change how it works locally, only moves everything to root on publish

**Comment by stubailo** [comment]
Yeah I suppose the import paths will be different for `npm link` though which is a bit odd... we'll figure it out later.

**Comment by stubailo** [comment]
Does the PR look good?

File: test/QueryManager.ts [hunk 1]
```diff
@@ -1379,15 +1385,44 @@ describe('QueryManager', () => {
...
+  it('warns if you forget the template literal tag', () => {
+    const queryManager = new QueryManager({
+      networkInterface: mockNetworkInterface(),
+      store: createApolloStore(),
+      reduxRootKey: 'apollo',
+    });
+
+    assert.throws(() => {
+      queryManager.query({
+        // Bamboozle TypeScript into letting us do this
```
Comment (mquandalle) [inline test/QueryManager.ts]: Maybe the trowing tests should be written in JS to avoid this kind of type system coercion?

Comment (stubailo) [inline test/QueryManager.ts]: Meh it's just one place right? I think it's great that typescript has a way of overriding types to allow us to test an edge case like this.

**Comment by stubailo** [comment]
Docs PR: https://github.com/apollostack/docs/pull/63

**Comment by jbaxleyiii** [comment]
@stubailo really excited for this. It looks great to me!

**Comment by stubailo** [comment]
Going for it!
